### PR TITLE
[MINI-2109] placeholder text overlapping search field

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
The instructional and placeholder text were overlapping due to iOS 11 different behavior.

Screen Shot 2020-11-12 at 12 30 36

## Links
[MINI-2109](https://jira.rakuten-it.com/jira/browse/MINI-2109)

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: MiniApp/Classes/MiniAppDownloader.swift [hunk 1]
```diff
@@ -32,6 +32,7 @@ class MiniAppDownloader {
         if isMiniAppAlreadyDownloaded(appId: appId, versionId: versionId) {
             if cacheVerifier.verify(appId: appId) {
                 let miniAppStoragePath = FileManager.getMiniAppVersionDirectory(with: appId, and: versionId)
+                MiniAppLogger.d("📂 \(miniAppStoragePath.absoluteString)")
```
Comment (Climbatize) [inline MiniApp/Classes/MiniAppDownloader.swift:35]: sorry for that, it's an artifact from another debug session

**Review by ts-leojoseph-r — CHANGES_REQUESTED** [review]
Looks like `js-miniapp` submodule is removed

**Comment by Climbatize** [comment]
> Looks like `js-miniapp` submodule is removed

😖

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**